common/changes/@typespec/compiler/deprecate-directive_2023-07-24-18-28.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@typespec/compiler",
+      "comment": "Add #deprecated directive",
```

Comment:
```suggestion
      "comment": "Add `#deprecated` directive",
```

Reply:
Updated the message as suggested and added a little more context